### DISCUSSION TITLE
Struggling to create my first security rule

### DISCUSSION THREAD

**Discussion by lucasnantonio** [supabase/supabase#1842] [Questions] [answered]
I'm new to Supabase and am struggling here. 
I have a table called `friends` with a `user` column, which references my ` auth users` table `id` with type `uuid`.

<img width="571" alt="CleanShot 2021-06-06 at 00 59 12@2x" src="https://user-images.githubusercontent.com/13240286/120913089-73649b80-c662-11eb-9e9b-892b828897e7.png">

But when I try to create my rule to allow users to update their own `friends` 
I get:
`operator does not exist: uuid = name`

<img width="374" alt="CleanShot 2021-06-06 at 01 01 06@2x" src="https://user-images.githubusercontent.com/13240286/120913131-b1fa5600-c662-11eb-97d7-448747ed01b6.png">

My rule
`auth.uid() = user`

<img width="649" alt="CleanShot 2021-06-06 at 01 02 03@2x" src="https://user-images.githubusercontent.com/13240286/120913159-d6eec900-c662-11eb-907d-ba2ae64b573e.png">

Any help?

**Comment by lucasnantonio** [comment, marked as answer]
Update: interestingly, changing the name of the column from `user` to `uid` solved the problem.

**Reply by easy-street** [reply]
`user` is a [PostgreSQL reserved key word](https://www.postgresql.org/docs/12/sql-keywords-appendix.html): it's never allowed as an identifier. for that reason it can be practical to avoid it altogether by choosing another name as you've already done. another option would be to quote the identifier:

```
auth.uid() = "user"
```

**Reply by lucasnantonio** [reply]
Thanks for clarifying 🙏
